www/js/lib/uiUtil.js

```diff
@@ -418,6 +418,8 @@ define(rqDef, function() {
      * @param {String} theme The theme to apply (light|dark[_invert|_mwInvert])
      */
     function applyAppTheme(theme) {
+        if (theme === "auto_invert") theme = window.matchMedia('(prefers-color-scheme:dark)').matches ? "dark_invert" : "light" 
+        if (theme === "auto_mwInvert") theme = window.matchMedia('(prefers-color-scheme:dark)').matches ? "dark_mwInvert" : "light" 
```

Comment:
Ditto about repeated code here. You don't need two tests. Just test the `auto` part as indicated above.

Reply:
I agree Mr. @Jaifroid , it would work in the above case, But in this case I am setting `theme` variable's value.

Reply:
OK, but you can still make the code more concise by defining `var darkPreference = window.matchMedia.....`, then do your tests on `darkPreference`